### PR TITLE
[16.0][FIX] account_move_name_sequence: computed fields raising warning

### PR DESCRIPTION
**Pull request by remi-filament**
Easy solution for #1614 by not computing related fields anymore thus warnings are not raised anymore for false gaps in sequence.

Still I guess we could find a way to preserve functionality to check if gap in sequence but cannot figure out how !

### CODE REVIEW

**Review by moylop260 — APPROVED** [review]
👍

**Comment by moylop260** [comment]
@luisg123v @hugho-ad @fernandahf

Please, review

**Review by fernandahf — APPROVED** [review]

**Comment by fernandahf** [comment]
LGTM

File: account_move_name_sequence/models/account_move.py [hunk 1]
```diff
@@ -9,11 +9,14 @@ class AccountMove(models.Model):
     _inherit = "account.move"
 
     name = fields.Char(compute="_compute_name_by_sequence")
-    # highest_name, sequence_prefix and sequence_number are not needed any more
+    # highest_name, sequence_prefix, sequence_number are not needed any more
```
Comment (luisg123v) [inline account_move_name_sequence/models/account_move.py:12]: I think semantic was better before

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Review by hugho-ad — APPROVED** [review]

**Review by alexis-via — APPROVED** [review]

**Review by RodrigoBM — APPROVED** [review]

**Pull request merged**